src/store/reducers/settings/types.ts

```diff
@@ -0,0 +1,23 @@
+import {changeFilter, SET_SETTING_VALUE} from './settings';
+
+export enum ProblemFilterValues {
```

Comment:
Lets try not to use enums (https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums)